finetuner/experiment.py

```diff
@@ -119,14 +122,20 @@ def create_run(
         model: str,
         train_data: Union[DocumentArray, str],
         run_name: Optional[str] = None,
+        eval_data: Optional[Union[DocumentArray, str]] = None,
+        csv_options: Optional[Dict[str, Any]] = None,
         **kwargs,
     ) -> Run:
         """Create a run inside the experiment.
 
         :param model: Name of the model to be fine-tuned.
```

Comment:
Lets completely remove this docstring args descriptions. We end up repeating ourselves all the time